nautobot_netbox_importer/diffsync/models/abstract.py

```diff
@@ -420,6 +441,7 @@ class ChangeLoggedModelMixin(BaseModel):
 
     # created is set automatically on model creation, so don't try to sync it between systems
```

Comment:
Mind removing this comment since it's no longer true? :-)

Reply:
yes, I did it as some point and forgot to commit